src/prisma/schema.prisma

```diff
@@ -76,20 +76,24 @@ model Tokens {
 }
 
 model WalletDetails {
-  address            String  @id @map("address")
-  type               String  @map("type")
-  label              String? @map("label")
+  address                            String  @id @map("address")
+  type                               String  @map("type")
+  label                              String? @map("label")
   // Local
-  encryptedJson      String? @map("encryptedJson")
+  encryptedJson                      String? @map("encryptedJson")
   // KMS
-  awsKmsKeyId        String? @map("awsKmsKeyId")
-  awsKmsArn          String? @map("awsKmsArn")
+  awsKmsKeyId                        String? @map("awsKmsKeyId") // deprecated and unused, todo: remove with next breaking change. Use awsKmsArn
+  awsKmsArn                          String? @map("awsKmsArn")
+  awsKmsSecretAccessKey              String? @map("awsKmsSecretAccessKey") // if not available, default to: Configuration.awsSecretAccessKey
+  awsKmsAccessKeyId                  String? @map("awsKmsAccessKeyId") // if not available, default to: Configuration.awsAccessKeyId
   // GCP
-  gcpKmsKeyRingId    String? @map("gcpKmsKeyRingId") @db.VarChar(50)
-  gcpKmsKeyId        String? @map("gcpKmsKeyId") @db.VarChar(50)
-  gcpKmsKeyVersionId String? @map("gcpKmsKeyVersionId") @db.VarChar(20)
-  gcpKmsLocationId   String? @map("gcpKmsLocationId") @db.VarChar(20)
-  gcpKmsResourcePath String? @map("gcpKmsResourcePath") @db.Text
+  gcpKmsKeyRingId                    String? @map("gcpKmsKeyRingId") @db.VarChar(50) // deprecated and unused. Use gcpKmsResourcePath instead, todo: remove with next breaking change
```

Comment:
Tip: You can use triple slash:
`/// @deprecated This is no longer tracked in DB.`

Prisma will generate it in typedocs for our editor:
<img width="526" alt="image" src="https://github.com/user-attachments/assets/30d91c58-57b3-45e4-92f1-dcca3baba5bd">

Reply:
Nice, very handy, thanks!